listenbrainz/webserver/views/index.py

```diff
@@ -120,6 +124,35 @@ def current_status():
     )
 
 
+@index_bp.route('/delete-user/<musicbrainz_id>')
+def mb_user_deleter(musicbrainz_id):
+    _authorize_mb_user_deleter(request.args.get('auth_token', ''))
+    delete_user(musicbrainz_id)
+    return jsonify({'status': 'ok'}), 200
+
+
+def _authorize_mb_user_deleter(auth_token):
+    query_url = 'https://musicbrainz.org/oauth2/userinfo'
```

Comment:
(I mentioned this in the BookBrainz PR too, but) it'd be nice if the MusicBrainz server URL were configurable, so we don't have to hand-edit source files when testing this locally (or against test.mb, say). Not just here, but for OAuth logins and all other references too.